Libplanet/Store/FileStore.cs

```diff
@@ -601,8 +632,112 @@ in GetStateReferences(stream))
             Block<T> branchPoint,
             IImmutableSet<Address> addressesToStrip)
         {
-            string sourceDir = GetStateReferencePath(sourceNamespace);
-            string targetDir = GetStateReferencePath(destinationNamespace);
+            ForkEntries(
```

Comment:
To me the term “entry” sounds too broad.  How about those alternative terms?

- appended index
- indexed stack

Reply:
I fixed <q>Entries</q> to <q>IndexedStack</q>.